### DISCUSSION TITLE
Classification mode with a point cloud

### DISCUSSION THREAD

**Discussion by clemencestr** [iTowns/itowns#1754] [Q&A] [answered]

**Comment by gchoqueux** [comment, marked as answer]
> Hello,
I use the example of the church but with my own data.
In the toggle menu there is a classification option (eglise_saint_blaise_arles -> Styling -> Display mode).
When I choose the classification mode, it displays all the points in gray.
Whereas in my .las my points have different classifications (2, 5...). There should be different colors displayed not only gray.
Do I need to do something in the code to activate the classification mode ?
Thanks

I believe that there is no classification in the church data

Normally, it is enough to use mode 2 in [`PointsMaterial`](http://www.itowns-project.org/itowns/docs/#api/Points/Classification)
to display the classifications.

Can you post a small laz file of your data?

**Reply by clemencestr** [reply]
I resolved my issue, the problem was the data, they hadn't classification...
Sorry for the inconvenience.
